content/docs/config-files.md

```diff
@@ -195,9 +195,32 @@ The `<WordsStyle>` `colorStyle` attribute decides whether to use the defined col
 * Set `colorStyle="0"`: this style will inherit both the foreground and background colors from the Default style (equivalent to right-clicking both the foreground and background colors in the UDL styler dialog box)
 
 
-## `functionList.xml`
+## `Function List`
 
-Defines what counts as a "function" for **View > Function List**.  There are some comments in the file, and lots of examples of the builtin languages, which you can customize.
+Defines what counts as a "function" for **View > Function List**.  There are some comments in the file(s), and lots of examples of the builtin languages, which you can customize.
+From v7.9.1 the file structure of function list parsers have been changed, so please follow one of sections below according your Notepad++ version.
+
+### v7.9.1 and later versions
+
+The `functionList` folder contains a separate XML file (function list parse rule) for each language's function list capability.
+Each function list parse rule links to a language with the language default name. For example the file name of php language parse rule is `php.xml`, the file name of Java language parse rule is `java.xml`, Check [overrideMap.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/functionList/overrideMap.xml) for the naming list of all supported programming languages.
+
+[overrideMap.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/functionList/overrideMap.xml) is optional. When you need to override a default parser rule or to define a function list parser for your User Defined Language (UDL), you modify this file.
+
+* Override a default parser rule examples
+```
+<association id= "anotherPhpParser.xml"			 langID= "1" />
+<association id= "myPerlRule.xml"			 langID= "21"/>
+```
+If 2 above lines are in *overrideMap.xml*, function list will load your parsers `anotherPhpParser.xml` and `myPerlRule.xml` instead of loaoding `php.xml` and `perl.xml` while showing PHP and Perl function list respectively.
+
+* Define your UDL example
+```
+<association id= "krl.xml"				userDefinedLangName="KRL"/>
+```
+Here you define a parser rule file name for your KRL UDL. While you open a file which is recognized as KRL file, then function list engin will load `functionList\krl.xml` to show the KRL function list. If you have no KRL UDL defined in your Notepad++, you have to define a dummy one (with the name "KRL") to make it work.
+
+### v7.9 and previous versions
 
 If you want to add **Function List** capability for your User Defined Language (UDL), you can.  You need to add two groups of information:

```

Comment:
Because we've split this up, the v7.9-and-earlier section no longer mentions the name of the config file.  We should insert a sentence at the beginning of this line:

> The `functionList.xml` config file contains XML entries for each language's function list definition, as well as a map that tells Notepad++ which section of the XML is applied to each file type.

Reply:
@pryrt 
You're right about it. I will add it.